Mackbellemore/soen-390-team07
# fix(client): memory leak and unneeded auth check call

#

## Changes this PR introduces

- Skip the auth check call if jwt is not set
- Fix memory leak warning in `useLoggedInUser` due to setting a state after the component has unmounted
- Remove usage of cookies and cookies package



## How to test this PR

- You shouldn't see a 403 status code from `authCheck` in the console when you click the login button











## Checklist
